java-iam/pom.xml

```diff
@@ -98,7 +98,7 @@
       <dependency>
         <groupId>com.google.cloud</groupId>
         <artifactId>google-iam-policy</artifactId>
-        <version>1.9.3-SNAPSHOT</version><!-- {x-version-update:google-iam-policy:current} -->
+        <version>1.10.0</version><!-- {x-version-update:google-iam-policy:current} -->
```

Comment:
@ddixit14  This line is managed by Release Please bot. Can you fix this?

Reply:
@ddixit14 Can you remove this google-iam-policy from this dependencyManagement section?